crates/shared/src/price_estimation/trade_verifier.rs

```diff
@@ -528,6 +530,7 @@ fn ensure_quote_accuracy(
         gas: summary.gas_used.as_u64(),
         solver,
         verified: true,
+        call_data: None,
```

Comment:
Shouldn't we store here the encoded call data that was used for the quote verification?
https://github.com/cowprotocol/services/blob/d9eb1894f6b829c9176c9fd46ddefda964d9e79f/crates/shared/src/price_estimation/trade_verifier.rs#L144-L152